### PR TITLE
Reword the introduction section of the specification

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: docs/spec/spec.md [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
 _Created_: 2021/12/12  
-_Updated_: 2021/12/12   
+_Updated_: 2022/02/10   
 _Issue_: [#2326](https://github.com/ballerina-platform/ballerina-standard-library/issues/2326)
 
 # Introduction
-This is the specification for the Task library which provides APIs to schedule a Ballerina job either once or periodically and to manage the execution of those jobs in the [Ballerina programming language](https://ballerina.io/), which is an open-source programming language for the cloud that makes it easier to use, combine, and create network services.
+This is the specification for the Task standard library of [Ballerina language](https://ballerina.io/), which provides APIs to schedule a Ballerina job either once or periodically and manage the execution of those jobs.
+
+The Task library specification has evolved and may continue to evolve in the future. Released versions of the specification can be found under the relevant Github tag.
```
Comment (daneshk) [inline docs/spec/spec.md:12]:
```suggestion
The Task library specification has evolved and may continue to evolve in the future. Released versions of the specification can be found under the relevant GitHub tag.
```

File: docs/spec/spec.md [hunk 2]
```diff
@@ -3,11 +3,17 @@
...
+
+If you have any feedback or suggestions about the library, start a discussion via a [Github issue](https://github.com/ballerina-platform/ballerina-standard-library/issues) or in the [Slack channel](https://ballerina.io/community/). Based on the outcome of the discussion, specification and implementation can be updated. Community feedback is always welcome. Any accepted proposal which affects the specification is stored under `/docs/proposals`. Proposals under discussion can be found with the label `type/proposal` in Github.
```
Comment (daneshk) [inline docs/spec/spec.md:14]:
```suggestion
If you have any feedback or suggestions about the library, start a discussion via a [GitHub issue](https://github.com/ballerina-platform/ballerina-standard-library/issues) or in the [Slack channel](https://ballerina.io/community/). Based on the outcome of the discussion, specification and implementation can be updated. Community feedback is always welcome. Any accepted proposal which affects the specification is stored under `/docs/proposals`. Proposals under discussion can be found with the label `type/proposal` in GitHub.
```

File: docs/spec/spec.md [hunk 3]
```diff
@@ -3,11 +3,17 @@
 _Owners_: @daneshk @kalaiyarasiganeshalingam                                      
 _Reviewers_: @daneshk  
 _Created_: 2021/12/12  
-_Updated_: 2021/12/12   
+_Updated_: 2022/02/10
```
Comment (shafreenAnfar) [inline docs/spec/spec.md:6]: We need to include the edition as well.

https://github.com/ballerina-platform/module-ballerina-http/blob/master/docs/spec/spec.md

File: docs/spec/spec.md [hunk 4]
```diff
@@ -3,11 +3,18 @@
...
```
Comment (shafreenAnfar) [inline docs/spec/spec.md:6]:
```suggestion
_Updated_: 2022/02/10  
```

**Review by daneshk — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**